### ISSUE TITLE
[a11y] `Magnifying glass` button in bottom navigation toolbar lacks tap ripple/highlight effect

### ISSUE THREAD

**Issue by hffvld** [brave/brave-browser#53007]
### Description

The `Magnifying glass` button in the bottom navigation toolbar lacks a tap ripple/highlight effect.

### Steps to reproduce

1. Launch Brave
2. Hamburger menu > Settings
3. Appearance > *Confirm that the bottom navigation toolbar is enabled*
4. Close the settings
5. Tap the `magnifying glass` button > *Observe that there is no ripple/highlight effect*

### Actual result

The `Magnifying glass` button in the bottom navigation toolbar lacks a tap ripple/highlight effect.

------------------

https://github.com/user-attachments/assets/3d3cbdd8-3ece-47ed-a317-e1bd8fb59e88

### Expected result

The `Magnifying glass` button in the bottom navigation toolbar must have a tap ripple/highlight effect similar to other buttons like `Home`, `Bookmark`, `Tab overview`, etc.

### Reproduces how often

Easily reproduced

### Brave version

Brave build: 1.89.56 
Chromium: 145.0.7632.109 (Official Build) canary (64-bit) 

### Device

- Pixel 6 / oriole-user 16 BP4A.251205.006 release-keys

### Channel information

- [x] release (stable)
- [x] beta
- [x] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome
